doc/release-notes/6.3-release-notes.md

```diff
@@ -0,0 +1,469 @@
+- Setting the query parameter `onlyDisplayedOnCreate=true` also returns metadata blocks with dataset field type input levels configured as required on the General Information page of the collection, in addition to the metadata blocks and their fields with the property ``displayOnCreate=true``.
+
+See also [#10389](https://github.com/IQSS/dataverse/pull/10389)
+
+dataverses/{id}/metadatablocks API endpoint has been fixed, since the fields returned for each metadata block when returnDatasetTypes query parameter is set to true was not correct. See #10637 and #10642.
+
+### Dataverse Payload Includes Release Status
+
+The Dataverse object returned by /api/dataverses has been extended to include "isReleased": {boolean}. See also [#10491](https://github.com/IQSS/dataverse/pull/10491).
+
+### New Field Type Input Level Endpoint
+
+A new endpoint ``api/dataverses/{id}/inputLevels`` has been created for updating the dataset field type input levels of a collection via API. See also [#10477](https://github.com/IQSS/dataverse/pull/10477).
+
+### Banner Message Endpoint Extended
+
+The endpoint `api/admin/bannerMessage` has been extended so the ID is returned when created. See also [#10565](https://github.com/IQSS/dataverse/pull/10565).
+
+[↑ Table of Contents](#table-of-contents)
+
+## Settings
+
+### Database Settings:
+
+***New:***
+
+- :DisableSolrFacets
+
+***Deprecated (used with rsync):***
+
+- :DataCaptureModuleUrl
+- :DownloadMethods
+- :LocalDataAccessPath
+- :RepositoryStorageAbstractionLayerUrl
+
+### New Configuration Options
+
+- `dataverse.files.<id>.disable-tagging`
+- `dataverse.feature.add-publicobject-solr-field`
+- `dataverse.feature.avoid-expensive-solr-join`
+- `dataverse.feature.reduce-solr-deletes`
+- `dataverse.feature.disable-return-to-author-reason`
+- `dataverse.feature.index-harvested-metadata-source`
+- `dataverse.ui.show-validity-label-when-published`
+
+[↑ Table of Contents](#table-of-contents)
+
+## Complete List of Changes
+
+For the complete list of code changes in this release, see the [6.3 Milestone](https://github.com/IQSS/dataverse/issues?q=milestone%3A6.3+is%3Aclosed) in GitHub.
+
+[↑ Table of Contents](#table-of-contents)
+
+## Getting Help
+
+For help with upgrading, installing, or general questions please post to the [Dataverse Community Google Group](https://groups.google.com/g/dataverse-community) or email support@dataverse.org.
+
+[↑ Table of Contents](#table-of-contents)
+
+## Upgrade Instructions
+
+Upgrading requires a maintenance window and downtime. Please plan accordingly, create backups of your database, etc.
+
+These instructions assume that you've already upgraded through all the 5.x releases and are now running Dataverse 6.2.
+
+0\. These instructions assume that you are upgrading from the immediate previous version. If you are running an earlier version, the only safe way to upgrade is to progress through the upgrades to all the releases in between before attempting the upgrade to this version.
```

Comment:
"only safe way" isn't really true - maybe "only supported way"?

Reply:
Sure, see by [Our experience on upgrading dataverse from 4.2.2 to 6.1 in a non-standard way](https://groups.google.com/g/dataverse-community/c/BUO37-reWIs/m/18SK0DZlAQAJ) by @wingfranky as an example of an alternative way of upgrading.

Reply:
I changed it to "supported" in 660b72b73d.